Request access restructuring

bnaolu opened the pull request:
Originally question 2, but will now be asked as question 1.
Also added some of the diction/structural changes from Philip's branch to this one.

Some thoughts that came up:
- I think the reordering helped somewhat, but personally, I still feel like this question is in its own bubble/category...let me know your thoughts 🤔

bnaolu committed:
Reordered original q2

Originally question 2, but will now be asked as the first question.

Also added some of the diction changes from Philip's branch.

pglevy commented:
As we discussed Tuesday, I think this simply "reorders" the issue instead of mitigating it. Maybe the "combined examples of reasons to file" is a better approach to test.

pglevy commented:
Also, I think it makes sense to keep the "who" question first, because even if you have a valid violation (a what), if it's not a covered entity, it won't matter anyway.

bnaolu commented:
> As we discussed Tuesday, I think this simply "reorders" the issue instead of mitigating it. Maybe the "combined examples of reasons to file" is a better approach to test.

Sounds good. Will get this approach up and running today.

bnaolu committed:
Question restructuring

- Restructured yaml file which now has only 3 questions total.
- Request to access is now worded as an option to select from in new Q2.
- Added content in accordion feature in new Q2.

Note: we may need to go back and clean up or remove some files (i.e. question-4.md, exit-what-1.md, and exit-what-2.md).

bnaolu committed:
Revert "Reordered original q2"

This reverts commit 3d8886b82d01ffcbb26c6d22ee7e0bc4972663d9.

bnaolu committed:
Merge pull request #32 from Bixal/revert-improve-flow-q2

Revert "Reordered original q2"
Reverted question order
